ci-operator/step-registry/openshift/e2e/vsphere/csi/openshift-e2e-vsphere-csi-workflow.yaml

```diff
@@ -16,8 +15,6 @@ workflow:
     - chain: gather-core-dump
     - chain: ipi-vsphere-post
     env:
-      # for storage-conf-feature-gate
-      FEATURESET: TechPreviewNoUpgrade
       # for storage-conf-wait-for-csi-driver
       CLUSTERCSIDRIVER: csi.vsphere.vmware.com
       TRUECONDITIONS: VMwareVSphereDriverControllerServiceControllerAvailable VMwareVSphereDriverNodeServiceControllerAvailable VMwareVSphereDriverStorageClassControllerAvailable
```

Comment:
I think these two vars can be removed too.

Reply:
removed